cloudsync/sync/manager.py

```diff
@@ -79,7 +79,7 @@ def download(self) -> str:
                     self.provider.download(self.info.oid, f)
                 os.rename(self.__temp_file + ".tmp", self.__temp_file)
             except Exception as e:
-                log.debug("error downloading %s", e)
+                log.error("error downloading %s", e)
```

Comment:
info or warning logging is sufficient for any issues that are related to "normal operation", but could be a provider, network, filesystem problem. 

probably we could have OSError be a warning, and Exception be an error... but I'm fine with warning on most things.

in the near future, errors will be actionable items